exporters/otlp/src/otlp_grpc_client.cc

```diff
@@ -23,6 +31,73 @@ namespace exporter
 namespace otlp
 {
 
+#ifdef ENABLE_ASYNC_EXPORT
+
+namespace
+{
+struct OPENTELEMETRY_LOCAL_SYMBOL OtlpGrpcAsyncCallDataBase
```

Comment:
I don't understand the use of OPENTELEMETRY_LOCAL_SYMBOL on a type.

I would expect this to apply to global variables, if any, not on a type definition.

Please clarify (or remove).

Reply:
This will hide vtable and functions when compiling with `-fvisibility=default`.